### PR TITLE
Add JSON Web Token (JWT) operation

### PR DESCRIPTION
**Pull request by licitdev**
Follow-up on #19332 with a JSON Web Token operation.

### Screenshots

image 
image 
image 
image

### CODE REVIEW

**Comment by jorgekorgut** [comment]
Amazing job!

File: .changeset/cold-pens-develop.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+"@directus/app": minor
+"@directus/api": minor
+---
+
+Added JSON Web Token (JWT) operation
```
Comment (paescuj) [inline .changeset/cold-pens-develop.md:6]:
```suggestion
Introduced new JSON Web Token (JWT) operation to Flows
```

**Review by paescuj — APPROVED** [review]
LGTM 🚀

**Comment by jaads** [comment]
Nicee 🎉

File: pnpm-lock.yaml [hunk 2]
```diff
@@ -1933,11 +1933,6 @@ importers:
 
 packages:
```
Comment (jaads) [inline pnpm-lock.yaml:1934]: it looks like all of the version changes below are downgrades - this is a bit weird isn't it? 🤔  probably not really important but I'm still wondering

Comment (licitdev) [reply pnpm-lock.yaml:1934]: Yeah! Noticed that but am not fully sure why that's the case. @paescuj had also tested and he created the exact same lockfile too. 🤔

Comment (paescuj) [reply pnpm-lock.yaml:1934]: We were wondering about the big diff as well, but it seems to be correct! I've regenerated the lockfile and it resulted into the same.

Comment (jaads) [reply pnpm-lock.yaml:1934]: alright 👍 should be fine then :)

Comment (paescuj) [reply pnpm-lock.yaml:1934]: One explanation could be that other dependencies are depending on `jsonwebtoken` as well, resulting in quite some changes when we update to the same or a different version.

Comment (jaads) [reply pnpm-lock.yaml:1934]: actually, when I update `jsonwebtoken` on the main branch, the `pnpm-lock` file does not change as much - just three lines changed 🤔

Comment (paescuj) [reply pnpm-lock.yaml]: Thanks for checking, indeed seems to behave differently on `main`. I've now merge in `main` and regenerated the lockfile so it should be up-to-date.

Comment (jaads) [reply pnpm-lock.yaml]: ah nice! now I can sleep better 😄 thanks! 🥳

**Review by jaads — APPROVED** [review]

**Pull request merged**